apps/rush-lib/src/logic/PublishGit.ts

```diff
@@ -86,13 +103,18 @@ export class PublishGit {
   }
 
   public commit(commitMessage: string): void {
-    PublishUtilities.execCommand(!!this._targetBranch, 'git', ['commit', '-m', commitMessage, '--no-verify']);
+    PublishUtilities.execCommand(!!this._targetBranch, this._gitPath, [
+      'commit',
```

Comment:
Personally I prefer this argument format, but please be consistent between using an array vs. `'...'.split(' ')`